### ISSUE TITLE
Error working with outlook date/time properties

### ISSUE THREAD

**Issue by RyanSylvestre** [idobatter/node-win32ole#2]
I am having trouble accessing properties such as LastModificationTime, Start and End on the appointment object in outlook. I am getting this error in the console when trying to access the item.Start / item.End

```
[Date (bug?)]
ASSERT in 00000000 module ..\src\v8variant.cc(91) @node_win32ole::V8Variant::CreateOCVariant: !v->IsUndefined()
error: The operation completed successfully.
```

I am able to work with the property in the restrict method on items so I think its a valid property.

``` javascript
var items = calendar.Items.restrict("[Start] > '03/10/2013'");
```

 I am looking to access the date as a string or something I can use in JS. Any thoughts?

**Comment by idobatter** [comment]
Hi,
Thank you so much for your report and fixed code ( https://github.com/RyanSylvestre/node-win32ole/commit/bef32729bbdbb379bef06f07c3165a4b50c4e1d2 ) .
I've merged it to new version win32ole@0.1.1 that supports Date type in JS.

**Comment by RyanSylvestre** [comment]
Thanks for the quick fix. It works great.
